plugins/ingest/src/main/java/org/elasticsearch/ingest/IngestDocument.java

```diff
@@ -155,6 +155,17 @@ public boolean hasField(String path) {
      * @throws IllegalArgumentException if the path is null, empty, invalid or if the field doesn't exist.
      */
     public void removeField(String path) {
+        removeField(TemplateService.wrap(path));
```

Comment:
I wonder what the final goal is here. Do we want to keep both variations 1) string argument 2) template argument or will the string one go away?